src/style.css

```
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url(images/star-active.svg);
}
```
join all selectors with the same styles